test-case/multiple-pipeline-capture.sh

```diff
@@ -137,7 +138,7 @@ do
     tmp_count=$(expr $tmp_count + $pcount)
     pcount=$(pidof aplay|wc -w)
     tmp_count=$(expr $tmp_count + $pcount)
-    [[ $tmp_count -ne $max_count ]] && func_error_exit "Target pipeline count: $max_count, current process count: $tmp_count"
+    [[ $tmp_count -ne $running_count ]] && func_error_exit "Target pipeline count: $running_count, current process count: $tmp_count"
```

Comment:
how to detail the "pipeline count" mismatch with parameter, I think here is conflict with test case comment